### ISSUE TITLE
[Accessibilty]A11y_AzureDataStudio_CSVModifyColumn_ScreenReader: NVDA does not announce correct order of the list item for “Data Type” Expanded list.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5963]
[Check out Keros!](
https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
#A11yMAS; #A11yTCS;  #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows10
NVDA Version: 2019.1
JAWS version:2019

**Additional Details:**
Screen reader name: NVDA
MAS reference: MAS1.3.1

**Repro Steps:**
1.Install the app Azure Data Studio.
2.Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->Click on next button.
3.Navigate to the table.
4.Select any list item from “Data Type” expand list and press enter key.
5.Observe that screen reader doesn’t read the ordering for list items under “Data Type”    expanded list.

**Actual:**
Screen reader announcing order for only 5 list items under “Data Type” expanded list and it reads only name of the remaining 29 list items.

**Same issue observed with Jaws.**

**Expected:**
Screen reader should announce order for all list items present under “Data type” expanded list.

**User Impact:**
Screen reader user is unable to understand order of list items are present on the page.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/default.asp

**MAS Reference:**
**MAS1.3.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[5963_[Accessibilty]A11y_AzureDataStudio_CSVModifyColumn_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3274831/5963_.Accessibilty.A11y_AzureDataStudio_CSVModifyColumn_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
Similar issue : https://github.com/microsoft/azuredatastudio/issues/6944
should have been fixed by this :https://github.com/microsoft/azuredatastudio/pull/7159

**Comment by Charles-Gagnon** [comment]
Possibly related to #5971 and #5969

**Comment by corivera** [comment]
I still see this issue with non-editable dropdowns with Narrator. The Command Palette also seems to have the same behavior. The announced order is based on the number of visible items in the dropdown. Since the max height of an editable dropdown is lower the issue occurs more often. In my testing I had a non-editable dropdown with 100 items, and it would only announce the entries as "X of 36" instead of 100, with 36 being the number of visible items.

CC @Benjin

**Comment by corivera** [comment]
Opened an issue in the vscode repo for this, since it affects all dropdowns. https://github.com/microsoft/vscode/issues/84306

**Comment by Charles-Gagnon** [comment]
In case this is useful later on if we have to do a fix on our end - the Tree constructor allows passing in an Accessibility provider that can add on the aria-posinset and aria-setsize attributes to the items in the tree. Just doing this wasn't working for some reason though so it appears some other work needs to be done (possible due to the roles assigned to the parent tree item)

https://github.com/microsoft/azuredatastudio/blob/master/src/sql/base/parts/editableDropdown/browser/dropdown.ts#L180
